staging/src/k8s.io/kms/scripts/setup-kind-cluster.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```

Comment:
The kind config already exists in https://github.com/kubernetes/kubernetes/tree/master/test/e2e/testing-manifests/auth/encrypt, so this script is not required. I think it's fine hardcoding the registry name and port in the kind config there and modifying it because we're using these manifests only for CI.